gap/examples.gi

```diff
@@ -711,6 +1435,110 @@ function(filt, m, n)
   return D;
 end);
 
+InstallMethod(WindmillGraphCons,
+"for IsImmutableDigraph, integer, integer",
+[IsImmutableDigraph, IsPosInt, IsPosInt],
+function(filt, n, m)
+  local D;
+  D := MakeImmutable(WindmillGraphCons(IsMutableDigraph, n, m));
+  SetIsMultiDigraph(D, false);
+  SetIsSymmetricDigraph(D, true);
+  SetChromaticNumber(D, n);
+  SetDigraphDiameter(D, 2);
+  return D;
+end);
+
+InstallMethod(WindmillGraph, "for two integers", [IsPosInt, IsPosInt],
+{n, m} -> WindmillGraphCons(IsImmutableDigraph, n, m));
+
+InstallMethod(WindmillGraph, "for a function and two integers",
+[IsFunction, IsPosInt, IsPosInt], WindmillGraphCons);
+
+InstallMethod(PathGraphCons, "for IsMutableDigraph and an integer",
+[IsMutableDigraph, IsPosInt],
+function(filt, n)
+  local D;
+  D := Digraph(IsMutableDigraph, []);
+  DigraphAddVertices(D, n);
+  DigraphAddEdges(D, List([1 .. (n - 1)], x -> [x, x + 1]));
+  return DigraphSymmetricClosure(D);
+end);
+
+InstallMethod(PathGraphCons,
+"for IsImmutableDigraph, integer",
+[IsImmutableDigraph, IsPosInt],
+function(filt, n)
+  local D;
+  D := MakeImmutable(PathGraphCons(IsMutableDigraph, n));
+  SetIsMultiDigraph(D, false);
+  SetIsSymmetricDigraph(D, true);
+  SetIsAcyclicDigraph(D, true);
+  SetIsEmptyDigraph(D, n = 1);
+  return D;
+end);
+
+InstallMethod(PathGraph, "for an integer", [IsPosInt],
+n -> PathGraphCons(IsImmutableDigraph, n));
+
+InstallMethod(PathGraph, "for a function and an integer",
+[IsFunction, IsPosInt], PathGraphCons);
```

Comment:
@LRacine has also implemented the same graph in #457 - I guess only one implementation is needed. You can decide amongst yourselves which one to keep!